src/components/TaskForm/TaskForm.jsx

```diff
@@ -218,25 +226,29 @@ export default function TaskForm() {
       });
   };
 
+  const handleDeleteTask = (taskId) => {
```

Comment:
nit: Common practice when code is not complete is to use "TODO : ...." 

if you have a task ID: "TODO (<task_id>): ...."